test/__init__.py

```diff
@@ -120,6 +120,7 @@ def assignment_response_data(self, decision='submit'):
             'response': 'response',
             decision: True,
             'amount': 2,
+            'duration': 5,
```

Comment:
This was needed to fix the test, because you added the duration to the response form the test must also send this information.